components/tools/OmeroWeb/test/unit/test_util.py

```diff
@@ -53,3 +57,20 @@ def test_get_date_time(self):
             getDateTime("invalid")
         with pytest.raises(ValueError):
             getDateTime("2015-12-01")
+
+    @pytest.mark.parametrize('top_links', [(
```

Comment:
I'm all for convenience and the use of `parametrize()` but could we write this out long hand so that it is self documenting, easier to see the intended outcome and easier to filter if one or more of these tests fails for some reason? It would also be good to understand how this affects failure conditions should someone input an incorrect value.